component/avatar/avatar.css

```diff
@@ -0,0 +1,95 @@
+.avatar_heading {
+    display: flex;
+    flex-wrap: wrap;
+    color: var(--third-color);
+    font-size: 4rem;
+    margin: 2rem 0rem 0rem 3rem;
+}
+
+.avatar_ex {
+    display: flex;
+    flex-wrap: wrap;
+    color: var(--third-color);
+    font-size: 2rem;
+    margin-left: 2rem;
+    margin-top: 2rem;
+}
+
+.para {
+    color: var(--secondary-color);
+    margin-left: 2rem;
+    margin-top: 1rem;
+    padding: 0rem;
+}
+
+.avatar {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    height: 20rem;
+    width: 100%;
+    margin-top: 2rem;
+}
+
+.avatar-image {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: flex-end;
+    flex-direction: row;
+    gap: 1rem;
+    border: 3px solid var(--avatar-boder-color);
+    border-radius: 0.5rem;
+    width: 70rem;
+    padding: 3rem;
+}
+
+.avatar_lg_size {
+    border: 1px solid black;
+    border-radius: 50%;
+    font-size: 5rem;
+    height: 12.5rem;
+    width: 12.5rem;
+}
+
+.avatar_medium_size {
+    border: 1px solid black;
+    border-radius: 50%;
+    font-size: 3rem;
+    height: 9.3rem;
+    width: 9.3rem;
+}
+
+.avatar_small_size {
+    border: 1px solid black;
+    border-radius: 50%;
+    font-size: 2rem;
+    height: 6rem;
+    width: 6rem;
+}
+
+.avatar_xs_size {
+    border: 1px solid black;
+    border-radius: 50%;
+    font-size: 1rem;
+    height: 3rem;
+    width: 3rem;
+
+}
+
+.avatar-text {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--primary-color);
+    color: white;
+}
+
+.primary-text-color {
+    color: var(--primary-color);
+}
+
+.avatarCode {
```

Comment:
Follow one convention for all the class names. According to your structure, this should be avatar_code.